seed-db/scripts/plan.js

```diff
@@ -44,7 +44,7 @@ const createPlan = (i, program, student) => {
         numYears: getRandomNumberInRange(1, 3),
         completed: false,
         createdAt: new Date().toISOString(),
-        updatedAT: new Date().toISOString(),
+        updatedAt: new Date().toISOString(),
```

Comment:
Does this field show up anywhere on the frontend?

Reply:
I think it does for something, makes the objetct stuff weird, i just accidentally named it wrong at the beginning